reactioncommerce/reaction-docs
# [WIP] docs : QA Matrix

Resolves #647 
Impact: **minor**  
Type: **docs**

## Issue
No consolidated testing matrix 

## Solution & Screenshots
Convert testing plan data from Sheets to MD to be visible in Testing area in the documentation with a live functionality

![image](https://user-images.githubusercontent.com/6117033/44687565-911b6580-aa1f-11e8-95fb-9a76e6f6b01c.png)

## Testing
1. Run `docker-compose up`
2. Check `localhost:4000/docs/next/testing-matrix`


More detail for what each of these sections should include are available in our [Contributing Docs](https://docs.reactioncommerce.com/reaction-docs/master/contributing-to-reaction)
